sympy/physics/continuum_mechanics/beam.py

```diff
@@ -867,7 +867,7 @@ def bending_moment(self):
         return integrate(self.shear_force(), x)
 
     def max_bmoment(self):
```

Comment:
This seems to already be merged, but it is an awful method name. It should be `max_bending_moment()` at least.